benchmark_app.go

```diff
@@ -66,6 +70,14 @@ func (c BenchmarkConfig) Validate() error {
 		return errors.New("must specify log filename")
 	}
 
+	if c.RateControlConfig.Connections != 0 {
+		if c.RateControlConfig.Concurrency == 0 {
+			return errors.New("must specify -concurrency if -connections is specified")
+		} else if (c.RateControlConfig.Connections % c.RateControlConfig.Concurrency) != 0 {
+			return errors.New("must specify -connections to be a multiple of -concurrency")
+		}
+	}
+
```

Comment:
So this validates the command line flags, and there are no validation in the WorkloadConfig.. hmm.

Reply:
Yes, I should also validate there, since they can also specify these values directly in WorkloadConfig.